integreat_cms/cms/migrations/0112_remove_poi_email_remove_poi_phone_number_and_more.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+from django.db import migrations
+
+if TYPE_CHECKING:
+    from django.apps.registry import Apps
+
+
+def create_primary_contact(apps: Apps) -> None:
+    """
+    Create primary contact from the poi contact data
+
+    :param apps: The configuration of installed applications
+    """
```

Comment:
```suggestion
if TYPE_CHECKING:
    from django.apps.registry import Apps
    from django.db.backends.base.schema import BaseDatabaseSchemaEditor


def create_primary_contact(
    apps: Apps,
    schema_editor: BaseDatabaseSchemaEditor,  # pylint: disable=unused-argument
) -> None:
    """
    Create primary contact for each POI and remove E-mail, phone number and website fields from POI model

    :param apps: The configuration of installed applications
    :param schema_editor: The database abstraction layer that creates actual SQL code
    """
```

To let the system work again, `schema_editor: BaseDatabaseSchemaEditor` is added.

The change in the documentation sentence is just a matter of preference, feel free to ignore if it's not appealing.

Reply:
I think your version is more explanatory and thank you for solving the schema error :heart_eyes: